### PR TITLE
Add 'news.test.files.bbci.co.uk' to non live csp

### LINKED ISSUE #7161: Add https://news.test.files.bbci.co.uk to our non-live csp
**Is your feature request related to a problem? Please describe.**
https://news.test.files.bbci.co.uk is not in our non-live csp for iframes. This means that some includes are not displaying on test. E.g. https://www.test.bbc.com/gahuza/23323415 where this error is seen:
image

**Describe the solution you'd like**
Add https://news.test.files.bbci.co.uk to our non-live csp for iframes, so that these includes can be tested on the test environment.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by oluoluoxenfree**
Resolves #7161 

**Overall change:**
Edit non-live iframe CSP.

**Code changes:**

- add new line to non-live CSPs

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by joshcoventry** [comment]
Someone else can correct me if I'm wrong, but I don't think we need to add anything to `src/server/utilities/cspHeader/index.js ` because this should only affect the test CSP.

**Comment by rhenshaw56** [comment]
> Someone else can correct me if I'm wrong, but I don't think we need to add anything to `src/server/utilities/cspHeader/index.js ` because this should only affect the test CSP.

@joshcoventry The canonical non-live CSP in `src/server/utilities/cspHeader/index.js`  covers for both local and test environments. In the past I think we had to also create a mozart pr to do these updates but that's since been updated to just let simorgh dictate it's csp and everything is housed in this file

**Review by rhenshaw56 — APPROVED** [review]

File: src/server/utilities/cspHeader/index.js [hunk 1]
```diff
@@ -114,6 +114,7 @@ const directives = {
       'https://www.youtube.com', // Social Embeds, <amp-youtube />
       'https://www.instagram.com', // Social Embeds, <amp-instagram />
       'https://*.ampproject.net', // Social Embeds
+      'https://news.test.files.bbci.co.uk',
```
Comment (rhenshaw56) [inline src/server/utilities/cspHeader/index.js]: I don't think we need this specified for amp this isn't shown as blocked on https://www.test.bbc.com/gahuza/23323415.amp

Comment (andrewscfc) [reply src/server/utilities/cspHeader/index.js]: I'm seeing an error on amp: 
image
The placeholder images are loading in place of the includes though:
image

**Review by andrewscfc — APPROVED** [review]

File: src/server/utilities/cspHeader/index.js (hunk 1 above)
Comment (andrewscfc) [reply src/server/utilities/cspHeader/index.js]: The placeholder image behaviour is expected but may give the impression the includes were loading correctly when they were not. I think we do need `https://news.test.files.bbci.co.uk` in the amp csp unless I'm missing something @rhenshaw56 ?

Comment (rhenshaw56) [reply src/server/utilities/cspHeader/index.js]: yeah, I get csp errors now. 

Screen Shot 2020-07-13 at 09 27 24

**Comment by staylos92** [comment]
LGTM 👍

**Pull request merged**